src/net.cpp

```diff
@@ -615,8 +613,7 @@ void CNode::GetBanned(std::map<CSubNet, int64_t> &banMap)
     banMap = setBanned; //create a thread safe copy
 }
 
-/// To be moved to CConnman
-bool CNode::IsWhitelistedRange(const CNetAddr &addr) {
+bool CConnman::IsWhitelistedRange(const CNetAddr &addr) {
```

Comment:
I guess for the follow-up to review data structures used here. Linear lookup?!

Reply:
Yes, agree. For this PR I did not change the underlying logic, and there is space for improvements